res/css/views/elements/_GenericEventListSummary.scss

```diff
@@ -80,7 +84,6 @@ limitations under the License.
 .mx_GenericEventListSummary_avatars {
     display: inline-block;
     margin-right: 8px;
-    padding-top: 8px;
```

Comment:
This can be removed because `mx_GenericEventListSummary_avatars` for bubble layout is set to zero on https://github.com/matrix-org/matrix-react-sdk/pull/8713/commits/fb696214ef769254fafc7eb43b2725a8aa5be648#diff-f5e6e48067f0db9e4d21def8a6998dc916f40442fe6618996b2484867dee705dR57